js/src/layers/SVGOverlay.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+const L = require('../leaflet.js');
+const rasterlayer = require('./ImageOverlay.js');
+
+const DEFAULT_LOCATION = [0.0, 0.0];
+
+export class LeafletSVGOverlayModel extends ImageOverlayModel {
+  defaults() {
+    return {
+      ...super.defaults(),
+      _view_name: 'LeafletSVGOverlayView',
+      _model_name: 'LeafletSVGOverlayModel',
+      svgElement: '',
```

Comment:
I guess it should be `el` instead of `svgElement` here? (`value` if you apply my suggestion above)

The comment applies later in the code for `this.model.get('svgElement')`